### PR TITLE
chore: delete the rest of chronograf

### LINKED ISSUE #20832: Remove `chronograf` folder in latest influxdb
some refactoring, but the chronograf folder at the root of the project is almost entirely unused. Tons of dead code there. we should remove it.

### PR DESCRIPTION
**Pull request by wbaker85**
Closes #20832

There were a few remaining things tangled in with chronograf, but after some digging, it turns out we can just delete those too!

This PR completes the removal of the chronograf folder.

### CODE REVIEW

File: http/influxdb/client.go [hunk 1]
```diff
@@ -19,37 +16,6 @@ var (
...
-	c.URL = url
-	c.Authorizer = DefaultAuthorization(s)
-	c.InsecureSkipVerify = s.InsecureSkipVerify
-	c.Logger = &chronograf.NoopLogger{}
-	return c, nil
-}
-
-// DefaultAuthorization creates either a shared JWT builder, basic auth or Noop
-// This is copy of the method from chronograf/influx adapted for platform sources.
-func DefaultAuthorization(src *platform.Source) influx.Authorizer {
```
Comment (wbaker85) [inline http/influxdb/client.go:37]: Wasn't used anywhere

File: http/influxdb/client.go [hunk 2]
```diff
@@ -19,37 +16,6 @@ var (
 	defaultTransport = &http.Transport{}
 )
 
-func newClient(s *platform.Source) (*influx.Client, error) {
```
Comment (wbaker85) [inline http/influxdb/client.go:22]: Only used in `http/influxdb/bucket.go`, which is getting deleted

File: http/influxdb/bucket.go [hunk 3]
```diff
@@ -1,80 +0,0 @@
...
-	"time"
-
-	platform2 "github.com/influxdata/influxdb/v2/kit/platform"
-
-	platform "github.com/influxdata/influxdb/v2"
-	"github.com/influxdata/influxdb/v2/kit/tracing"
-)
-
-// BucketService connects to Influx via HTTP using tokens to manage buckets
-type BucketService struct {
```
Comment (wbaker85) [inline http/influxdb/bucket.go:15]: There sure are a lot of bucket services in this application! Fortunately, this one wasn't used anywhere else.

File: source/bucket.go [hunk 4]
```diff
@@ -1,30 +0,0 @@
...
-	"fmt"
-
-	platform "github.com/influxdata/influxdb/v2"
-	"github.com/influxdata/influxdb/v2/http"
-	"github.com/influxdata/influxdb/v2/http/influxdb"
-	"github.com/influxdata/influxdb/v2/tenant"
-)
-
-// NewBucketService creates a bucket service from a source.
-func NewBucketService(s *platform.Source) (platform.BucketService, error) {
```
Comment (wbaker85) [inline source/bucket.go:13]: This function was used in the launcher to build the `http/api_handler.go` `APIBackend` struct, but it was never used anywhere after that.

**Review by danxmoran — APPROVED** [review]
🎉

**Pull request merged**